src/guides/v2.3/config-guide/prod/config-reference-var-name.md

```diff
@@ -242,7 +242,7 @@ content='
 
 *  To use values you set in the `$_ENV` array, you must set `variables_order = "EGPCS"` in your `php.ini` file. For details, see [PHP documentation](http://us.php.net/manual/en/ini.core.php#ini.variables-order).
 
-*  For Magento Commerce Cloud, if you are attempting to override Magento configuration settings using the [Project Web Interface](https://devdocs.magento.com/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
+*  For {{ site.data.var.ece }}, if you are attempting to override Magento configuration settings using the [Project Web Interface](https://devdocs.magento.com/cloud/project/project-webint-basic.html#project-conf-env-var), you must prepend the variable name with `env:`. For example:
```

Comment:
Revert this change. Liquid variables are not allowed within Liquid tags. This one is within  `{% include note.html ... %}`.